sample/host.json

```diff
@@ -12,5 +12,8 @@
     },
     "queues": {
         "visibilityTimeout": "00:00:10"
+    },
+    "swagger": {
+        "isEnabled": true
```

Comment:
Let's change this to `enabled` to be consistent with the other boolean flags we have (e.g. `excluded`, `disabled`, etc.)